core/src/main/scala/org/locationtech/rasterframes/util/package.scala

```diff
@@ -183,6 +184,27 @@ package object util {
     }
   }
 
+  implicit class DFWithPrettyPrint(val df: Dataset[_]) extends AnyVal {
+    def toMarkdown(numRows: Int = 5, truncate: Boolean = false): String = {
+      import df.sqlContext.implicits._
+      val cols = df.columns
+      val header = cols.mkString("| ", " | ", " |") + "\n" + ("|---" * cols.length) + "|\n"
+      val stringifiers = cols
+        .map(c => s"`$c`")
+        .map(c => df.col(c).cast(StringType))
+        .map(c => if (truncate) substring(c, 1, 40) else c)
+      val cat = concat_ws(" | ", stringifiers: _*)
+      val body = df
+        .select(cat).limit(numRows)
+        .as[String]
+        .collect()
+        .map(_.replaceAll("\\[", "\\\\["))
+        .map(_.replace('\n', '↩'))
```

Comment:
Is this a necessity to avoid breaking the table formatting? As a reader, it's a little unusual to see.